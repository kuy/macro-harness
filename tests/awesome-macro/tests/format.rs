pub fn test() {
    format!("answer = {}", 42);
}
