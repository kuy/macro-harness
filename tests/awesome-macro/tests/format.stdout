#![feature(prelude_import)]
#[prelude_import]
use std::prelude::rust_2018::*;
#[macro_use]
extern crate std;
pub fn test() {
    {
        let res = ::alloc::fmt::format(::core::fmt::Arguments::new_v1(
            &["answer = "],
            &match (&42,) {
                (arg0,) => [::core::fmt::ArgumentV1::new(
                    arg0,
                    ::core::fmt::Display::fmt,
                )],
            },
        ));
        res
    };
}
